test/mpi/f08/datatype/createf90.f90

```diff
@@ -1,68 +0,0 @@
-!
-! Copyright (C) by Argonne National Laboratory
-!     See COPYRIGHT in top-level directory
-!
-
-        program main
-        use mpi
-        integer ierr
-        integer errs
-        integer nints, nadds, ndtypes, combiner
-        integer nparms(2), dummy(1)
-        integer (kind=MPI_ADDRESS_KIND) adummy(1)
-        integer ntype1, nsize, ntype2, ntype3, i
-!
-!       Test the Type_create_f90_xxx routines
-!
-        errs = 0
-        call mtest_init( ierr )
-
-! integers with up to 9 are 4 bytes integers; r of 4 are 2 byte,
-! and r of 2 is 1 byte
-        call mpi_type_create_f90_integer( 9, ntype1, ierr )
```

Comment:
there is a F08 binding for this function in the standard. should we update this test to use it?

Reply:
There is already an updated version of this test called createf08.f90

Reply:
There is already an updated version of this test called createf08.f90